src/app/public/modules/inline-delete/inline-delete.component.html

```diff
@@ -0,0 +1,41 @@
+<div
+  class="sky-inline-delete sky-inline-delete-{{type}}"
+  role="alertdialog"
+  [attr.aria-describedby]="assistiveTextId"
+  [attr.aria-label]="'skyux_inline_delete_confirm_deletion' | skyLibResources"
+>
+  <span
+   class="sky-inline-delete-assistive-text"
+   [id]="assistiveTextId"
+  >
+    {{ 'skyux_inline_delete_assistive_text' | skyLibResources }}
+  </span>
+  <div *ngIf="pending"
+    class="sky-inline-delete-wait"
+  >
+    <sky-wait
+      [isWaiting]="true"
+    ></sky-wait>
+  </div>
+  <div
+    class="sky-inline-delete-content"
+    [@skyAnimationEmerge]="animationState"
+    (@skyAnimationEmerge.done)="onAnimationDone($event)"
+  >
+    <button
+      autofocus
```

Comment:
The `autofocus` doesn't seem to be working in any of the visuals. Should this also have a unit test?

Reply:
Fixed. I did have to use typescript for this due to `autofocus` not seeming to work on all browsers in this case. I think it might have partially be due to the animation but don't know 100&. It oddly did work in IE but not Chrome *sigh*.